torch/fx/experimental/symbolic_shapes.py

```diff
@@ -268,6 +307,9 @@ def eval(cls, base, divisor):
                 return FloorDiv(
                     sympy.simplify(base / gcd), sympy.simplify(divisor / gcd)
                 )
+            else:
+                return sympy.floor(base / divisor)
```

Comment:
I'm not sure we want to actually do this if the base and divisor are integer; I think I would rather keep the floor div unevaluated in this case

Reply:
Done, but note that I also had to update `evaluate_expr` because of it. We can no longer short-circuit there. Otherwise, an uneval'd `FloorDiv` will propagate and will fail somewhere else. E.g., in `SymNode.guard_float` when it tries to do `float(FloorDiv(...))`.